aws-exposed-key-checker-infra/lambda_source/exposed_key_checker/ticket_manager.py

```diff
@@ -45,13 +48,42 @@ def get_tickets_from_api(
         tickets = [TicketData.from_dict(t) for t in res["results"]]
         return tickets, res["next_page"]
 
+    def set_ticket_as_solved(self, ticket: "TicketData"):
+        print(f"Ticket to set to solved: {ticket.id}")
+
+        url = f"{self._api_base_url}/api/v2/tickets/update_many.json?ids={ticket.id}"
+
+        headers = {"Content-Type": "application/json"}
+
+        payload = {
+            "ticket": {
+                "status": "solved",
+                "assignee_id": ZENDESK_ASSIGNEE,
+                "comment": {
+                    "body": "Resolved by AWS key checker lambda.",
+                    "public": True,
+                },
+                "additional_tags": ["closed-by-aws-exposed-key-checker"],
+            }
+        }
+
+        data = json.dumps(payload)
+
+        response = requests.put(url, headers=headers, data=data, auth=self._auth)
```

Comment:
```suggestion
        response = requests.put(url, headers=headers, json=payload, auth=self._auth)
```

We can skip the manual json dump.

Reply:
good point, done